src/main/scala/io/moia/scalaHttpClient/HttpClient.scala

```diff
@@ -27,7 +27,7 @@ class HttpClient(
       httpMetrics,
       retryConfig,
       clock,
-      Logger.takingImplicit(LoggerFactory.getLogger(getClass.getName))((msg: String, _: NoLoggingContext) => msg),
+      Logger.takingImplicit(LoggerFactory.getLogger(classOf[HttpClient].getName))((msg: String, _: NoLoggingContext) => msg),
```

Comment:
`Predef.getClass` was unreliable all the time, so it was removed in Scala `2.13.2`, see https://github.com/scala/bug/issues/12025.